prov/efa/src/efa_base_ep.c

```diff
@@ -185,6 +185,10 @@ int efa_qp_create(struct efa_qp **qp, struct ibv_qp_init_attr_ex *init_attr_ex)
 			efa_attr.flags |= EFADV_QP_FLAGS_UNSOLICITED_WRITE_RECV;
 #endif
 		efa_attr.driver_qp_type = EFADV_QP_DRIVER_TYPE_SRD;
+#if HAVE_EFADV_SL
+		if (tclass == FI_TC_LOW_LATENCY)
+			efa_attr.sl = SL_8;
```

Comment:
Notice that if service level isn't supported by the device, ```-EINVAL``` will be returned by ```efadv_create_qp_ex``` for any non zero value.

Reply:
Do you have any comments on that service level value macro? Are we having something similar in rdma-core?